ext/drawText/.gitignore

```diff
@@ -0,0 +1,7 @@
+Build/
+Logs
+Makefile
+ModuleCache.noindex/
+drawText.bundle
+info.plist
+mkmf.log
```

Comment:
All these were generated by the `compile` Rake task. Because they're runtime generated, I think it's safer to ignore them.